src/pages/ProductList/common/ListSidebar.js

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from 'react';
+import FlexibleBtn from '../components/FlexibleBtn';
+import styled from 'styled-components';
+import {
+  faWonSign,
+  faLocationDot,
+  faStar,
+  faMap,
+  faFilter,
+  faList,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { motion } from 'framer-motion';
+
+const ListSidebar = props => {
+  const { hotelData, setView } = props;
+
+  const icon = [faList, faMap, faFilter];
+  const [showFilter, setShowFilter] = useState(false);
+
+  const [price, setPrice] = useState([]);
+
+  const [wonValue, setWonValue] = useState(0);
+  const [locationValue, setLocationValue] = useState(0);
+  const [starValue, setStarValue] = useState(1);
```

Comment:
```suggestion
  const [showFilter, setShowFilter] = useState(false);
  const [price, setPrice] = useState([]);
  const [wonValue, setWonValue] = useState(0);
  const [locationValue, setLocationValue] = useState(0);
  const [starValue, setStarValue] = useState(1);
  
  const icon = [faList, faMap, faFilter];
```

이렇게 있어야 순서가 맞겠습니다.
hooks의 자리는 함수의 top level입니다.
중간에 state끼리 공백도 줄여주시는게 가독성에 좋습니다

Reply:
확인했습니다 감사합니다